_i18n/en.yml

```diff
@@ -484,20 +484,23 @@ mining:
+  leading_para1: The majority of existing @cryptocurrencies, including Bitcoin and Ethereum, have transparent @blockchains; transactions can be verified and/or traced by anyone in the world. This means that the sending and receiving @addresses of these @transactions could potentially be linked to real-world identities.
+  leading_para2: Monero, on the other hand, uses various privacy-enhancing technologies to ensure the anonymity of its users. 
   confidential: Monero transactions are confidential and untraceable.
-  confidential_para1: Every Monero transaction, by default, obfuscates sending and receiving addresses as well as transacted amounts. This always-on privacy means that every Monero user's activity enhances the privacy of all other users, unlike selectively transparent cryptocurrencies (e.g. Zcash).
-  confidential_para2: Monero is fungible. By virtue of obfuscation, Monero cannot become tainted through participation in previous transactions. This means Monero will always be accepted without the risk of censorship.
+  confidential_para1: "Unlike selectively transparent alternatives (e.g. Zcash), Monero is the only cryptocurrency where every user is anonymous by default. The sender, receiver, and amount of every single transaction is hidden through the use of three important technologies: @Stealth-Addresses, @Ring-Signatures, and @Ring-Confidential-Transactions."
+  confidential_para2: Because every transaction is private, Monero cannot be traced. This makes it a true, @fungible currency; merchants and individuals accepting Monero do not need to worry about blacklisted or tainted coins.
   grassroots: Monero is a grassroots community attracting the world's best cryptocurrency researchers and engineering talent.
-  grassroots_para1: Over
-  grassroots_para2: 500 developers
-  grassroots_para3: have contributed to the Monero project, including 30 core developers. Forums and chat channels are welcoming and active.
-  grassroots_para4: Monero's Research Lab, Core Development Team and Community Developers are constantly pushing the frontier of what is possible with cryptocurrency privacy and security.
-  grassroots_para5: Monero is not a corporation. It is developed by cryptography and distributed systems experts from all over the world that donate their time or are funded by community donations. This means that Monero can't be shut down by any one country and is not constrained by any particular legal jurisdiction.
+  grassroots_para1: In addition to its 30 core developers, over
+  grassroots_para2: 500 community developers
+  grassroots_para3: have contributed to the Monero project. Numerous
+  grassroots_para4: forums and chat channels 
+  grassroots_para5: are available for anyone to join, and the community is always welcoming new members.
+  grassroots_para6: Monero's Research Lab
+  grassroots_para7: ", Core Development Team, and Community Developers are at the forefront of cryptocurrency privacy and security."
```

Comment:
Please remove the comma at the beginning and add it to the string before. A comma at the beginning of the string can be easily missed by translators.